9control.tex

```diff
@@ -1199,7 +1199,7 @@ \subsubsection{Recruitment Deviation Setup}
 	1971 \Tstrut & Main recruitment deviations begin year & If begin year is less than the model start year, then the early deviations are used to modify the initial age composition. However, if set to be more than the population maximum age before start year, it is changed to equal to the maximum age before start year. \Bstrut\\
 	\hline
 
-	2017 \Tstrut & Main recruitment deviations end year & If recruitment deviations end year is later than retro year, it is reset to equal retro year. The final year to estimate main recruitment deviations should be set to a year where information about young fish in the data becomes limited. As example, if the model end year is 2020 and the fleet/survey only starts observing fish of age 2+, the last year to estimate main recruitment deviations could be set to 2018. Years after the main period but before the end model year will be estimated as late deviations \Bstrut\\
+	2017 \Tstrut & \hypertarget{RecDevEndYear}{Main recruitment deviations end year} & If recruitment deviations end year is later than retro year, it is reset to equal retro year. The final year to estimate main recruitment deviations should be set to a year where information about young fish in the data becomes limited. As example, if the model end year is 2020 and the fleet/survey only starts observing fish of age 2+, the last year to estimate main recruitment deviations could be set to 2018. Years after the main period but before the end model year will be estimated as late deviations \Bstrut\\
```

Comment:
I think we should revise the text order here a bit since it seems weird to start with the special condition about retrospectives. I would move that first sentence to actually be the last sentence here.